nucypher/characters/control/controllers.py

```diff
@@ -178,27 +71,60 @@ def __init__(self,
         # Internals
         self._transport = None
 
-        # Hard-wire the character's output flow to the Emitter
-        self._internal_controller = character_controller
-        self._internal_controller.emitter = self.emitter
+        self.interface = interface
+
+        def set_method(name):
+
+            def wrapper(request=None, **kwargs):
+                request = request or kwargs
```

Comment:
This is a shim for optionally passing kwargs to a controller rather than a dictionary (unsupported behaviors), which needs to be more formally enforced.